Java/Java.sublime-syntax

```diff
@@ -598,20 +573,56 @@ contexts:
 
   parens:
     - match: \(
+      scope: punctuation.section.parens.begin.java
       push:
+        - meta_scope: meta.parens.java
```

Comment:
I'm not quite sure, tbh. Python also uses `group`, where contents in parentheses are just evaluated before others, pretty much like Java.

Maybe @wbond can clarify.